src/sinks/datadog/logs/sink.rs

```diff
@@ -102,8 +95,8 @@ pub struct LogSink<S> {
     acker: Acker,
     /// The API service
     service: S,
-    /// The encoding of payloads
-    encoding: EncodingConfigFixed<DatadogLogsJsonEncoding>,
```

Comment:
Note that this will remove the `encoding.{only_fields,except_fields,timestamp_format}` options and therefore be a breaking change.